packages/asgardex-bitcoin/src/client.ts

```diff
@@ -34,7 +44,7 @@ class Client implements BitcoinClient, AsgardexClient {
   nodeApiKey = ''
 
```

Comment:
[nitpick] `geAddress()` ([line 29](https://github.com/Pusher-Labs/asgardex-clients/blob/893f00867579c657f1f2ec12a54917d4a0dcdef9/packages/asgardex-bitcoin/src/client.ts#L29)) can be removed from `interface BitcoinClient`, it's [already define](https://github.com/Pusher-Labs/asgardex-clients/blob/master/packages/asgardex-client/src/types.ts#L83)d in `interface AsgardexClient`